com.github.jerempw.gnonograms.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.github.jeremypw.gnonograms",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.jeremypw.gnonograms",
+
+    "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--filesystem=home",
+      "--device=dri"
+    ],
+
+    "build-options": {
+        "cflags": "-O2 -g -w",
+        "cxxflags": "-O2 -g"
+    },
+
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+
+    "modules": [
+        {
+            "name": "libgranite",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/elementary/granite.git",
+                    "tag": "5.5.0"
+                }
+            ]
+        },
```

Comment:
use elementary baseApp (look at other elementary apps on Flathub for examples)

Reply:
OK, thanks.